FAIMS/FAIMS3
# feat: survey list home page

## JIRA Ticket

[BSS-129](https://jira.csiro.au/browse/BSS-129)

## Description

Update home page to be the survey list page. 

## Proposed Changes

- updated survey list to be on "/" route
- removed current component on "/" route
- updated all links to survey list to the "/" route

## How to Test

- log in to app
- home page should be survey list page
- try navigating away and back by clicking the logo/sidebar link
- log out of the app
- try navigating to http://localhost:3000/ and verify it redirects to the sign in page

## Additional Information

## Checklist
